Code/Framework/AzToolsFramework/AzToolsFramework/Application/ToolsApplication.cpp

```diff
@@ -17,6 +17,8 @@
 #include <AzFramework/TargetManagement/TargetManagementComponent.h>
 #include <AzFramework/StringFunc/StringFunc.h>
 
+#include <AzNetworking/Framework/NetworkingSystemComponent.h>
+
```

Comment:
This include can be removed once the AzNetworkingModule is setup to configure the NetworkingSystemComponent as required.